.github/workflows/test_rust.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
     - name: Install linux depencencies
       if: matrix.os == 'ubuntu-latest'
-      run: sudo apt-get -y install libasound2-dev
+      run: sudo apt-get -y install libasound2-dev libxcb-shape0-dev libxcb-xfixes0-dev
```

Comment:
We should probably document these requirements in the readme - but, we haven't done that for the existing `libasound2-dev` either :D